packages/core/src/components/Steps/StepsHeader.tsx

```diff
@@ -15,9 +15,9 @@ export interface StepsHeaderProps extends VibeComponentProps {
   onChangeActiveStep: (e: React.MouseEvent, stepIndex: number) => void;
   stepsCount: number;
   areNavigationButtonsHidden: boolean;
-  backButtonProps: ButtonProps;
-  nextButtonProps: ButtonProps;
-  finishButtonProps: ButtonProps;
+  backButtonProps: Partial<ButtonProps>;
+  nextButtonProps: Partial<ButtonProps>;
+  finishButtonProps: Partial<ButtonProps>;
```

Comment:
I changed it in a different PR since Button's children is now mandatory but I think making it a Partial is a better solution instead of providing `children: ''` as it already have defaults